std_msgs/Float64 lat # latitude in degrees
std_msgs/Float64 lon # longitude in degrees
std_msgs/Float32 z #usually depth, or meters below ocean surface
std_msgs/UInt8 z_units # 0=None,1=Depth,2=Altitude,3=Height
std_msgs/Float32 speed
std_msgs/UInt8 speed_units # 0=Meters per second, 1=RPM 
